include/xsimd/config/xsimd_config.hpp

```diff
@@ -244,6 +244,17 @@
 #define XSIMD_WITH_AVX512DQ 0
 #endif
 
+/**
```

Comment:
should be in an independent commit, preferably with a test case that catches the scenario we missed.